src/FindTranslocations.cpp

```diff
@@ -15,165 +15,47 @@
 #include <iostream>
 #include <fstream>
 
-
-
 #include "data_structures/Translocation.h"
+#include "data_structures/extract.cpp"
+#include "data_structures/findTranslocationsOnTheFly.cpp"
```

Comment:
In general you `#include` only the .h files, that contain the headers only. There are several ways to do this, among the most important keep the implementation separated from the interface (have a look [here](http://www.cplusplus.com/forum/articles/10627/)  ). Moreover it gives you speed up at compilation time (if you include the cpp it means that you are coping and pasting the code, and therefore recompiling it every time you make a change to FindTranslocation.cpp)